src/Flex.php

```diff
@@ -104,11 +103,13 @@ public function activate(Composer $composer, IOInterface $io)
 
         $rfs = Factory::createRemoteFilesystem($this->io, $this->config);
         $this->rfs = new ParallelDownloader($this->io, $this->config, $rfs->getOptions(), $rfs->isTlsDisabled());
+        $repositoryClassesProperty = new \ReflectionProperty(RepositoryManager::class, 'repositoryClasses');
+        $repositoryClassesProperty->setAccessible(true);
         $manager = new RepositoryManager($this->io, $this->config, $composer->getEventDispatcher(), $this->rfs);
```

Comment:
An alternative solution would be to use `RepositoryFactory::manager()` here to create the `RepositoryManager` instance, but that would not take into account any changes that could have been made (by plugins, for example) to the original manager.